### PR TITLE
Refactoring plist to html

### PR DESCRIPTION
**Pull request by csordasmarton**

### CODE REVIEW

File: tools/plist_to_html/tests/libtest/env.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import os
+import tempfile
+
+
+def test_proj_root():
+    return os.path.abspath(os.environ['TEST_PROJ'])
+
+
+def get_workspace(test_id='test'):
+    tmp_dir = os.path.join(os.environ['REPO_ROOT'], 'build')
```
Comment (gyorb) [inline tools/plist_to_html/tests/libtest/env.py]: Shouldn't we handle the newly introduced `CC_TEST_WORKSPACE_ROOT` env var here too?

File: tools/plist_to_html/tests/libtest/project.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# -----------------------------------------------------------------------------
```
Comment (gyorb) [inline tools/plist_to_html/tests/libtest/project.py]: Do we really need this extra module just to have a `path` function?

File: tools/plist_to_html/tests/projects/notes/notes.cpp [hunk 3]
```diff
@@ -0,0 +1,15 @@
+void log();
```
Comment (gyorb) [inline tools/plist_to_html/tests/projects/notes/notes.cpp]: Please add license section.

File: tools/plist_to_html/tests/libtest/env.py (hunk 1 above)
Comment (csordasmarton) [reply tools/plist_to_html/tests/libtest/env.py]: I've created a separate env var for this: `PLIST_TO_HTML_TEST_WORKSPACE_ROOT`

**Pull request merged**